llvm/test/CodeGen/WebAssembly/simd-shift-in-loop.ll

```diff
@@ -9,19 +9,21 @@ target triple = "wasm32-unknown-unknown"
 define void @shl_loop(ptr %a, i8 %shift, i32 %count) {
```

Comment:
A bit unfortunate that this regresses, but I think we'll recover this in the future (after https://github.com/llvm/llvm-project/pull/102133 + inference for gep nuw in instcombine, at which point we should be able to preserve the nuw flag here, which is all wasm cares about).